nepalingo-web/src/index.css

```diff
@@ -1,3 +1,23 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
\ No newline at end of file
+  @tailwind base;
```

Comment:
can you make sure that lint runs on this file - the spacing looks a bit off

Reply:
I don't think the lint catches formatting issues like this, but try to get rid of the extra spaces in this file.
On a related note, there are also lint issues that you need to fix. Error that appear when you run the command `pnpm run lint`. You can also see the errors in github actions like so:

![CleanShot 2024-07-08 at 22 05 27](https://github.com/nepalcodes/nepalingo/assets/12178490/f38c8aba-9f57-4a0e-b178-16395ebd8304)

Fix any errors that appear for your files, you can ignore the errors that appear related to `useNewari`.